packages/suite/src/views/wallet/staking/components/SolStakingDashboard/components/Rewards/RewardsList.tsx

```diff
@@ -0,0 +1,182 @@
+                                                        values={{ count: SOLANA_EPOCH_DAYS }}
+                                                    />
+                                                }
+                                            >
+                                                <Badge size="small">
+                                                    <Row gap={spacings.xxs} alignItems="center">
+                                                        <Translation
+                                                            id="TR_STAKE_REWARDS_BAGE"
+                                                            values={{ count: reward.epoch }}
+                                                        />
+                                                        <Icon name="info" size="small" />
+                                                    </Row>
+                                                </Badge>
+                                            </Tooltip>
+                                        </Column>
+                                    </Row>
+                                    {reward?.amount && (
+                                        <Column alignItems="end">
+                                            <HiddenPlaceholder>
+                                                <FormattedCryptoAmount
+                                                    value={formatNetworkAmount(
+                                                        reward?.amount,
+                                                        account.symbol,
+                                                    )}
+                                                    symbol={account.symbol}
+                                                />
+                                            </HiddenPlaceholder>
+                                            <HiddenPlaceholder>
+                                                <Text typographyStyle="hint" variant="tertiary">
+                                                    <FiatValue
+                                                        amount={formatNetworkAmount(
+                                                            reward?.amount,
+                                                            account.symbol,
+                                                        )}
+                                                        symbol={account.symbol}
+                                                    />
+                                                </Text>
+                                            </HiddenPlaceholder>
+                                        </Column>
+                                    )}
+                                </Row>
+                            </Card>
+                        </React.Fragment>
+                    ))}
+                </>
+            )}
+
+            {showPagination && (
```

Comment:
pagination is rendered faster than the components above 

Uploading Screen Recording 2025-02-04 at 21.18.19.mov…